src/controllers/check.your.answers.controller.ts

```diff
@@ -1,10 +1,36 @@
 import { Request, Response } from "express";
 import * as config from "../config";
 import { logger } from "../utils/logger";
+import { ApplicationData } from "../model";
+import { getApplicationData } from "../utils/application.data";
+import { presenterRole } from "../model/presenter.model";
 
 export const get = (req: Request, res: Response) => {
   logger.debug(`GET ${config.CHECK_YOUR_ANSWERS_PAGE}`);
+
+  const appData: ApplicationData = getApplicationData(req.session);
   return res.render(config.CHECK_YOUR_ANSWERS_PAGE, {
-    backLinkUrl: config.MANAGING_OFFICER_CORPORATE_URL
+    backLinkUrl: config.MANAGING_OFFICER_CORPORATE_URL,
+    appData,
+    presenterRole: getPresenterRoleDisplayText(appData)
   });
 };
+
+const getPresenterRoleDisplayText = (appData: ApplicationData): string | undefined => {
```

Comment:
Why this function, the check can be done on the template makes things more readable

Reply:
If you want to use `Nunjucks` I would have done something like 
```
{% set presenterRole %}
    {% if appData.presenter && appData.presenter.role %}
        {% if appData.presenter.roleTitle === "administrator" %}
            "Administrator"
        {% elif appData.presenter.roleTitle === "agent" %}
            "Agent"
        {% elif appData.presenter.roleTitle === "solicitor" %}
            "Solicitor"
        {% elif appData.presenter.roleTitle === "beneficialOwner" %}
            "Beneficial owner"
        {% elif appData.presenter.roleTitle %}
            appData.presenter.roleTitle
        {% endif %}
    {% endif %}
{% endset -%}
```